src/careamics_restoration/config/data.py

```diff
@@ -31,8 +24,6 @@ class SupportedExtensions(str, Enum):
 
     TIFF = "tiff"
     TIF = "tif"
-    NPY = "npy"  # TODO remove numpy after we reupload the dataset as zarr
```

Comment:
Is there any reason to remove it now, while it's still present in the dataset class ?

Reply:
You caught me, I really wanted to get rid of this. But makes no sense because we are still using it! 

Added back here: https://github.com/CAREamics/careamics-restoration/pull/59/commits/150bbc2cf9aa7ea743bd717924b03fd52c0d330f